haskell-pkgs.nix

```diff
@@ -1,18 +1,20 @@
 let
   # Fetch the latest haskell.nix and import its default.nix
-  haskellNix = import (builtins.fetchTarball{
-    url = "https://github.com/input-output-hk/haskell.nix/archive/f6663a8449f5e4a7393aa24601600c8f6e352c97.tar.gz";
-  }) {};
+  haskellNix = import
+    (builtins.fetchTarball {
+      url = "https://github.com/input-output-hk/haskell.nix/archive/1b9b05beed75a1be98405501fbb33ac1f080069e.tar.gz";
+    })
+    { };
 
-# haskell.nix provides access to the nixpkgs pins which are used by our CI,
-# hence you will be more likely to get cache hits when using these.
-# But you can also just use your own, e.g. '<nixpkgs>'.
-  nixpkgsSrc = haskellNix.sources.nixpkgs-2003;
+  # haskell.nix provides access to the nixpkgs pins which are used by our CI,
+  # hence you will be more likely to get cache hits when using these.
+  # But you can also just use your own, e.g. '<nixpkgs>'.
+  nixpkgsSrc = haskellNix.sources.nixpkgs-2009;
```

Comment:
iohk's hydra does not cache GHC-8.10.4 that was build with nixpkgs-2003, see https://input-output-hk.github.io/haskell.nix/reference/supported-ghc-versions/